src/main/java/Duke.java

```diff
@@ -1,10 +1,255 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static final String HORIZONTAL_LINE = "____________________________________________________________";
+
     public static void main(String[] args) {
```

Comment:
Method could be too long, exceeding 30 lines of code, it might be better to shorten it or refactor out certain portions to other methods.